src/lib_c/x86_64-windows-msvc/c/winnt.cr

```diff
@@ -6,4 +6,6 @@ lib LibC
   alias WCHAR = UInt16
   alias LPSTR = CHAR*
   alias LPWSTR = WCHAR*
+  alias LPCWSTR = WCHAR*
```

Comment:
`LPCWSTR` shouldn't be an alias since it's just constant which means nothing ABI-wise.

Reply:
So... what should it be?

Reply:
`LPWSTR`...